[2025-04-11T07:06:26.994+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: proyecto_bigdata.procesar_datos manual__2025-04-11T07:06:22.683039+00:00 [queued]>
[2025-04-11T07:06:27.002+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: proyecto_bigdata.procesar_datos manual__2025-04-11T07:06:22.683039+00:00 [queued]>
[2025-04-11T07:06:27.003+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-04-11T07:06:27.016+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): procesar_datos> on 2025-04-11 07:06:22.683039+00:00
[2025-04-11T07:06:27.023+0000] {standard_task_runner.py:60} INFO - Started process 431 to run task
[2025-04-11T07:06:27.026+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'proyecto_bigdata', 'procesar_datos', 'manual__2025-04-11T07:06:22.683039+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/proyecto_bigdata_dag.py', '--cfg-path', '/tmp/tmp7blny461']
[2025-04-11T07:06:27.028+0000] {standard_task_runner.py:88} INFO - Job 8: Subtask procesar_datos
[2025-04-11T07:06:27.084+0000] {task_command.py:423} INFO - Running <TaskInstance: proyecto_bigdata.procesar_datos manual__2025-04-11T07:06:22.683039+00:00 [running]> on host 4193653a407b
[2025-04-11T07:06:27.170+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='proyecto_bigdata' AIRFLOW_CTX_TASK_ID='procesar_datos' AIRFLOW_CTX_EXECUTION_DATE='2025-04-11T07:06:22.683039+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-04-11T07:06:22.683039+00:00'
[2025-04-11T07:06:27.200+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-04-11T07:06:27.209+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=proyecto_bigdata, task_id=procesar_datos, execution_date=20250411T070622, start_date=20250411T070626, end_date=20250411T070627
[2025-04-11T07:06:27.241+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-04-11T07:06:27.262+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
